apps/aehttpserver/web/controllers/info_controller.ex

```diff
@@ -6,6 +6,7 @@ defmodule Aehttpserver.InfoController do
   alias Aecore.Utils.Blockchain.BlockValidation
   alias Aecore.Keys.Worker, as: Keys
   alias Aecore.Peers.Worker, as: Peers
+  alias Aecore.Peers.Scheduler, as: Scheduler
```

Comment:
we don't need this

Reply:
We have missed those. A quick fix is on its way.